xlb/operator/stepper/nse_stepper.py

```diff
@@ -220,7 +220,9 @@ def kernel3d(
             feq = self.equilibrium.warp_functional(rho, u)
 
             # Apply collision
-            f_post_collision = self.collision.warp_functional(f_post_stream, feq, rho, u)
+            f_post_collision = self.collision.warp_functional(
+                f_post_stream, feq, rho, u
+            )
```

Comment:
Please don't use the formatter in your PRs where the focus is not the formatting.

Reply:
Merge this. I am setting up ruff for the project. The reason of this change was the ruff was not respected in previous PRs.